Vagrantfile

```diff
@@ -69,26 +69,27 @@ Vagrant.configure('2') do |config|
 
   bin_path = File.join(ANSIBLE_PATH_ON_VM, 'bin')
 
-  if Vagrant::Util::Platform.wsl? || (Vagrant::Util::Platform.windows? and !Vagrant.has_plugin? 'vagrant-winnfsd')
+  vagrant_mount_type = vconfig.fetch('vagrant_mount_type')
+
+  if vagrant_mount_type != 'nfs' || Vagrant::Util::Platform.wsl? || (Vagrant::Util::Platform.windows? && !Vagrant.has_plugin?('vagrant-winnfsd'))
+    vagrant_mount_type = nil if vagrant_mount_type == 'nfs'
```

Comment:
`nfs` is still a valid `type`. Why not just set it to that instead of this?

Reply:
@swalkinshaw this needs to be set to `nil` when on a Windows platform without support for nfs to allow the vagrant driver to choose type automatically.